src/settings_gui.cpp

```diff
@@ -579,7 +579,7 @@ struct GameOptionsWindow : Window {
 				break;
 
 			case WID_GO_VIDEO_DRIVER_INFO:
-				SetDParamStr(0, VideoDriver::GetInstance()->GetInfoString());
+				SetDParamStr(0, std::string{VideoDriver::GetInstance()->GetInfoString()});
```

Comment:
Might it be useful to create a `SetDParamStr` that accepts `std::string_view` and then does this wrapping into `std::string`? That way we have one location that misuses this trick, instead of it being spread over the code. It will also make other uses of `std::string_view` easier.

Reply:
If I remember correctly, the issue with that is overloading between `const char *`, `std::string` and `std::string_view` breaks things.